src/pages/proposal/completed/GrantProgram.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo } from 'react';
+import { Cancel, CheckCircle } from '@mui/icons-material';
+import { ListItemText, Stack, Typography } from '@mui/material';
+import type { Applicant, Proposal, VotingOption } from '@/types';
+import StateButton from '@/components/StateButton';
+import { toPastTense } from '@/helpers/string';
+
+interface GrantProgramProps {
+  data: Proposal;
+  voteTypeName?: string;
+}
+const GrantProgram = ({ data, voteTypeName }: GrantProgramProps) => {
+  const votedList: VotingOption[] = useMemo(
+    () =>
+      data.options.filter(opt =>
+        data.voted?.find(v => v.option === opt.option)
+      ),
+    [data.voted]
+  );
+
+  function isApplicant(value: unknown): value is Applicant {
+    return typeof value !== 'string';
+  }
+
+  return (
+    <Stack
+      direction="row"
+      spacing="12px"
+      marginRight="24px"
```

Comment:
use theme defined baseline instead of specific size, i.e., `spacing={1.5}` and `marginRight={3}`